app/jobs/action_cable_broadcast_job.rb

```diff
@@ -1,5 +1,5 @@
 class ActionCableBroadcastJob < ApplicationJob
-  queue_as :default
+  queue_as :critical
```

Comment:
@sojan-official This job has the highest throughput. Should this be `critical`?

Reply:
@vishnu-narayanan, the time to execute the job is minimal, and that's the one job we want to get processed near real-time.  we can switch this around if we observe any issues.